web/src/components/tasks/business-formation/BusinessFormationPaginator.test.tsx

```diff
@@ -691,70 +688,6 @@ describe("<BusinessFormationPaginator />", () => {
             expect(page.getStepStateInStepper(LookupStepIndexByName(formationStepName))).toEqual("ERROR");
             expect(screen.getByText(Config.formation.errorBanner.incompleteStepsError)).toBeInTheDocument();
           });
-
-          it("shows API error message on step for businessName API error", async () => {
```

Comment:
dust - these tests were for the error above the section right? The one the story wants to remove, ergo we don't need them anymore? (Just want to double check)

Reply:
yes, that's correct.